src/content/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring.mdx

```diff
@@ -9,109 +9,85 @@ translate:
+    </figcaption>
 
-<figcaption>
-  The <DoNotTranslate>**Hosts**</DoNotTranslate> page for your infrastructure shows performance indicators for systems, networks, processes, and storage.
-</figcaption>
-
-<Callout variant="tip">
-Want to learn how to use infrastructure monitoring effectively? Try out our guided [infrastructure and root causes lab](/docs/infrastructure/infrastructure-monitoring/get-started/identify-root-causes-guide).
-</Callout>
+## One agent, many capabilities [#overview]
 
-## Our infrastructure monitoring solutions [#infrastructure-solutions]
+To get access to our infrastructure monitoring capability, you need the infrastructure agent. When you install the agent, it can collect performance and health data about system resources and processes for on-premise or virtualized hosts. From there, you can configure the agent to pair with one of our supported [on-host integrations](/docs/integrations/host-integrations/getting-started/introduction-host-integrations), or to [forward logs](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent).
 
-We have a wide range of infrastructure monitoring solutions. Here's a summary:
+    <img
+    title="Agents and New Relic"
+    alt="Agents and New Relic"
+    src={infrastructureAngetsandNr}
+    />
 
-* Our [infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent) for Linux, macOS, and Windows operating systems.
-* Our [Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure) and [Pixie](/docs/kubernetes-pixie/auto-telemetry-pixie/get-started-auto-telemetry-pixie) integrations.
-* Our [cloud integrations](/docs/infrastructure/infrastructure-integrations/get-started/introduction-infrastructure-integrations/#cloud-integrations) for Amazon, Azure, and Google Cloud Platform.
-* Our [Prometheus integrations](/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic).
-* Our [on-host integrations](/docs/infrastructure/infrastructure-integrations/get-started/introduction-infrastructure-integrations/#on-host-integrations) for services like MySQL, NGINX, Cassandra, and Kafka (to name a few). Most of the on-host integrations are installed with the infrastructure agent and work in concert with it to send data.
+Infrastructure monitoring empowers modern operations teams to dig deep into complex and hybrid systems, from a single data center to thousands of Amazon, Google Cloud, or Azure instances. Whatever your architecture looks like, our infrastructure monitoring capability can help you troubleshoot and improve your systems. Here are some solutions:
```

Comment:
Again, I'd cut "can" from "can help you" and just make it "helps you". "Can do XYZ" is always too wishy-washy for me.

Reply:
hahahah yes. sometimes i like wishy washy b/c i dont want to make guarantees, but ur prob right in 90% of cases we can (heh) just go full active